appengine-standard-java8/translate-pubsub/src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <threadsafe>true</threadsafe>
+  <runtime>java8</runtime>
+  <application>YOUR-PROJECT-ID</application>
+
+  <env-variables>
+    <env-var name="PUBSUB_TOPIC" value="your-topic" />
```

Comment:
Fine as is, but could be set from the command line w/ mvn to make testing easier.

Reply:
There is a step to do that in the README for running locally, but you still need the env var for deploying, right?

Reply:
You can put `value="${myTopic}" />`, and use `mvn clean appengine:deploy -DmyTopic=xyzzy` to make deployment testing easier.